doc/source/examining/Loading_Generic_Particle_Data.ipynb

```diff
@@ -24,84 +20,84 @@
     "ppx, ppy, ppz = 1e6*np.random.normal(size=[3, n_particles])\n",
     "\n",
     "ppm = np.ones(n_particles)"
-   ]
+   ],
+   "outputs": [],
+   "metadata": {
+    "collapsed": false
+   }
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
    "source": [
     "The `load_particles` function accepts a dictionary populated with particle data fields loaded in memory as numpy arrays or python lists:"
-   ]
+   ],
+   "metadata": {}
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {
-    "collapsed": false
-   },
-   "outputs": [],
    "source": [
     "data = {'particle_position_x': ppx,\n",
     "        'particle_position_y': ppy,\n",
     "        'particle_position_z': ppz,\n",
     "        'particle_mass': ppm}"
-   ]
+   ],
+   "outputs": [],
+   "metadata": {
+    "collapsed": false
+   }
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
    "source": [
     "To hook up with yt's internal field system, the dictionary keys must be 'particle_position_x', 'particle_position_y', 'particle_position_z', and 'particle_mass', as well as any other particle field provided by one of the particle frontends."
-   ]
+   ],
+   "metadata": {}
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
    "source": [
     "The `load_particles` function transforms the `data` dictionary into an in-memory yt `Dataset` object, providing an interface for further analysis with yt. The example below illustrates how to load the data dictionary we created above."
-   ]
+   ],
+   "metadata": {}
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {
-    "collapsed": false
-   },
-   "outputs": [],
    "source": [
     "import yt\n",
     "from yt.units import parsec, Msun\n",
     "\n",
     "bbox = 1.1*np.array([[min(ppx), max(ppx)], [min(ppy), max(ppy)], [min(ppz), max(ppz)]])\n",
     "\n",
-    "ds = yt.load_particles(data, length_unit=parsec, mass_unit=1e8*Msun, bbox=bbox)"
-   ]
+    "ds = yt.load_particles(data, length_unit=1*parsec, mass_unit=1e8*Msun, bbox=bbox)"
```

Comment:
I honestly find it surprising that this fixed it!  I did not think the old way would be an error.  Does it also work to use `length_unit = "pc"`?

Reply:
Yup it works with "pc". The nicer fix may actually be for `load_particles` to accept units instead of quantities, by implicitly converting `parsec` to `1*parsed`. Wdyt?

Reply:
The bad news is that now the next cell crashes

Reply:
@AshKelly or any SPH-dataset guru to the rescue!